src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+
+    result: 0,
+
+    add(a) {
+      calculator.result += a;
+
+      return calculator.result;
+    },
```

Comment:
```suggestion
   add(a) {
      this.result += a;

      return this;
    },
```

use this keyword instead calculator
same for other operations methods